dumpcap: add page (#8042)
tldr-pages/tldr: pages/common/dumpcap.md

# dumpcap

> A network traffic dump tool.
> More information: <https://www.wireshark.org/docs/man-pages/dumpcap.html>.

> - Get a list of available interfaces:

Reviewer: ```suggestion
- Display available interfaces:
```

`dumpcap --list-interfaces`

> - Start capturing packets on the first interface:

Reviewer: ```suggestion
- Capture packets on a specific interface:
```

> `dumpcap --interface 1`

Reviewer 2: ```suggestion
`dumpcap --interface {{1}}`
```

> - Start capturing packets on the first interface to a specific location:

Reviewer: ```suggestion
- Capture packets to a specific location:
```

> `dumpcap --interface 1 -w {{path/to/output_file.pcapng}}`

Reviewer 2: ```suggestion
`dumpcap --interface {{1}} -w {{path/to/output_file.pcapng}}`
```

> - Start capturing packets on the first interface to a specific location and writing to a ring buffer with a max of 10 files of 500MB each:

Reviewer: ```suggestion
- Write to a ring buffer with a specific max file limit of a specific size:
```

> `dumpcap --interface 1 -w {{path/to/output_file.pcapng}} --ring-buffer filesize:500000 --ring-buffer files:10`

Reviewer 2: ```suggestion
`dumpcap --interface {{1}} -w {{path/to/output_file.pcapng}} --ring-buffer filesize:{{500000}} --ring-buffer files:{{10}}`
```

> # dumpcap - Dump network traffic

Reviewer 3: This should be the command name
```suggestion
# dumpcap
```

> A network traffic dump tool.

> > Preferably one line; two are acceptable if necessary.

Reviewer 3: :eyes: 
```suggestion
```

> More information: <https://www.wireshark.org/docs/man-pages/dumpcap.html>.

- Get a list of available interfaces:

> `dumpcap -D`

Reviewer 3: We prefer long arguments if available
```suggestion
`dumpcap --list-interfaces`
```

- Start capturing packets on the first interface:

> `dumpcap -i 1`

Reviewer 3: ```suggestion
`dumpcap --interface 1`
```

- Start capturing packets on the first interface to a specific location:

> `dumpcap -i 1 -w path/to/write/file/to.pcapng`

Reviewer 3: ```suggestion
`dumpcap --interface 1 -w {{path/to/output_file.pcapng}}`
```

- Start capturing packets on the first interface to a specific location and writing to a ring buffer with a max of 10 files of 500MB each:

> `dumpcap -i 1 -w path/to/write/file/to.pcapng -b filesize:500000 -b files:10`

Reviewer 3: ```suggestion
`dumpcap --interface 1 -w {{path/to/output_file.pcapng}} --ring-buffer filesize:500000 --ring-buffer files:10`
```
